actors/common/src/main/java/org/sunbird/learner/actors/OrganisationManagementActor.java

```diff
@@ -1873,4 +1880,68 @@ private void validateLocationCodeAndIds(Map<String, Object> request) {
   private boolean isEventSyncEnabled() {
     return Boolean.parseBoolean(getEventSyncSetting(JsonKey.ORGANISATION));
   }
+
+
+  private Map<String, Object> enquireOrg(String id) {
+    Map<String, Object> responseMap = new HashMap<>();
+    Response response = cassandraOperation.getRecordById(orgDbInfo.getKeySpace(), orgDbInfo.getTableName(), id);
+    Map<String, Object> record = response.getResult();
+    if (null != record && null != record.get(JsonKey.RESPONSE)) {
+      if(((List) record.get(JsonKey.RESPONSE)).size()!=0) {
+        responseMap = (Map<String, Object>) ((List) record.get(JsonKey.RESPONSE)).get(0);
+      }
+      ProjectLogger.log(
+              "OrganisationManagementActor:enquireOrg found org with Id: "+id, LoggerEnum.ERROR.name());
+    }
+    return responseMap;
+  }
+
+  private void assuringAssignKeysOperation(String orgId) {
```

Comment:
assuringAssignKeysOperation to isRootOrgIdValid and it will return boolean

Reply:
Acknowledged and Executed